pandas/core/arraylike.py

```diff
@@ -367,7 +367,7 @@ def reconstruct(result):
         if method == "__call__":
```

Comment:
```suggestion
        if method == "__call__" and kwargs.get("out", None) is None:
```

(untested, but if

Reply:
Or maybe more simply check for any `kwargs`, so only do this if `kwargs` is empty (so `and not kwargs`)

Reply:
I quickly pushed this edit myself, as it fixes the xfail case in the tests you added

Reply:
Thanks for adding this!